src/routes/RepresentativeRoute/RepresentativeRoute.js

```diff
@@ -59,53 +67,100 @@ export default class RepresentativeRoute extends React.Component {
+            {url && (
+              <p className="repPage-url">
+                <span className="repPage-span">Website</span>{' '}
+                <span className="site">
+                  <a href={url} target="_blank" rel="noopener noreferrer">
+                    Visit
+                  </a>
+                </span>
+              </p>
+            )}
+            {twitterUrl && (
+              <a
+                href={twitterUrl}
+                className="social-icon"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <Icon name="twitter" />
+              </a>
+            )}
+            {fbUrl && (
+              <a
+                href={fbUrl}
+                className="social-icon"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <Icon name="facebook" />
+              </a>
+            )}
+            <Link className="back-nav" to="/dashboard">
               {' \u2b05 Go Back'}
             </Link>
           </div>
           <div className="repPage-section-other repPage-section-image">
             <div className="representativeImage">{currentRepImg}</div>
           </div>
         </section>
-        {contribs ? (<>
-          <section className="repPage-section">
-            <TextContributions contributions={topContribs}/>
-            <h3 className="chartDesc">
-              This chart shows the top 5 contributors to your representative and
-              how much money your representative accepted from each one.
-            </h3>
-            <FinancialContributions contributions={topContribs} />
-          </section>
-          <section className="repPage-section">
-            <TextContributions contributions={topIndustries}/>
-            <h3 className="chartDesc">
-              This chart shows the top 5 business sectors that your representative
-              takes donations from and how much comes from each one.
-            </h3>
-            <FinancialContributions contributions={topIndustries} />
-          </section>
-          <section className="repPage-section">
-            <TextContributions contribs={contribs}/>
-            <h3 className="chartDesc">
-              This chart shows the total amount of money your representative took
-              in donations and how much they spent in the last cycle.
-            </h3>
-            <TotalContributions contribs={contribs} />
-          </section>
-        </>)
-        : <MoneySpinner />
-      }
+        {contribs ? (
+          <>
+            <section className="repPage-section">
+              <TextContributions contributions={topContribs} />
+              <h3 className="chartDesc">
+                Among all organizations, your representative accepted the
+                greatest dollar amount from these organizations. They may
+                possess a disproportionate influence over your representative's
```

Comment:
Yeah somewhat haha, though it’s also somewhat common knowledge. I wanted to err away from merely summarizing/restating the chart because that doesn’t add much value. Wanted to be somewhat forceful about why this info could be valuable. I also was trying to be careful using the “may” language but I’m definitely open to changing anything. Don’t want to seem biased

Reply:
I don't see bias.  It's not just one side that is swayed by special interests.